### ISSUE TITLE
Nessa nova versão não tem o index do Certificado Digital?

### ISSUE THREAD

**Issue by ieleny** [nfephp-org/sped-common#172]
E se não tiver posso usar o modelo da antiga versão?

**Comment by robmachado** [comment]
O que ??? index de certificado ???

**Comment by ieleny** [comment]
Sim, na antiga versão tinha uma tela de index.  Que preenchia os campos de acordo com o certificado. No projeto que foi depreciado.

**Comment by robmachado** [comment]
Acho que você se refere ao INSTALL !!
Não tem como usar aquilo nessa versão, erá para outra realidade.
Nessa versão você pode manter seu certificado onde quiser, em disco local, na nuvem em base de dados.
O que tem que fazer é ler o certificado e passar o conteúdo do PFX para a classe Certificate::class

**Comment by ieleny** [comment]
isso
